README.md

```diff
@@ -21,6 +21,8 @@ any changes to node-level settings before applying the new profile. The
 containerized TuneD daemon handles termination signals by rolling back any
 node-level settings it has applied before gracefully shutting down.
 
+## Perfomance Profile Controller
+[Performance Addon controller](docs/performance_addon.md), previously known as Performance addons operator and now a part of the Node Tuning Operator, optimizes OpenShift clusters for applications sensitive to cpu and network latency.
```

Comment:
Can we keep formatting at least reasonably consistent in this `README.md`?  I try (an sometimes fail) to wrap (old school) at 80 characters.  The line presented here is way too long.